tests/test_benchmark_simple_scenarios.py

```diff
@@ -261,6 +258,49 @@ def test_benchmark_AE_grid_battery_peak_pricing(self, margs):
                 ):
                     assert abs(battery_charge[i - 1]) > 0
 
+    @pytest.mark.skipif(
+        EXECUTE_TESTS_ON not in (TESTS_ON_MASTER),
+        reason="Benchmark test deactivated, set env variable "
+        "EXECUTE_TESTS_ON to 'master' to run this test",
+    )
+    @mock.patch("argparse.ArgumentParser.parse_args", return_value=argparse.Namespace())
+    def test_benchmark_AFG_grid_heatpump_heat(self, margs):
+        use_case = "AFG_grid_heatpump_heat"
+        main(
+            overwrite=True,
+            display_output="warning",
+            path_input_folder=os.path.join(TEST_INPUT_PATH, use_case),
+            input_type=CSV_EXT,
+            path_output_folder=os.path.join(TEST_OUTPUT_PATH, use_case),
+        )
+        # read json with results file
+        with open(
+            os.path.join(TEST_OUTPUT_PATH, use_case, "json_with_results.json"), "r"
+        ) as results:
+            data = json.load(results)
```

Comment:
using `load_json` will provide you directly with the timeseries in the dict (no need to read the xls file then)

Reply:
yes this will be tackled in #514